tests/post/steps/test_backup.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+from pytest_bdd import scenario, when, then, parsers
+
+from tests import utils
+
+# Scenarios
+@scenario("../features/backup.feature", "Backup multiple times")
+def test_backup(host):
+    pass
+
+
+# When
+@when(parsers.parse("we run the backup script {times} times"))
+def run_backup(request, host, times):
+    iso_root = request.config.getoption("--iso-root")
+
+    for _ in range(int(times)):
+        with host.sudo():
+            res = host.run("%s/backup.sh", str(iso_root))
+            assert res.rc == 0, res.stdout
+
+    with host.sudo():
+        res = host.run(
+            "kubectl --kubeconfig=/etc/kubernetes/admin.conf "
+            "wait --for=condition=complete --timeout=60s "
+            "job -l app.kubernetes.io/name=backup-replication --namespace=kube-system"
+        )
+        assert res.rc == 0, res.stdout
```

Comment:
I would prefer to move this to another "when" so that the process described in the feature is closer to what a user will do "when I run the backup 6 times and wait for the replication job to complete then I only have 5 backups"